### PR TITLE
Chrome 51 Deprecations and Removals.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: src/content/en/updates/posts/2016/04/chrome-51-deprecations.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+description: "An round up of the deprecations and removals in Chrome to help you plan."
+published_on: 2016-04-20
+updated_on: 2016-04-20
+authors:
+  - josephmedley
+tags:
+  - deprecations
+  - removals
+  - chrome51
+featured_image: /web/updates/images/2016/04/chrome-51-deprecations/deps-removes.png
```
Comment (PaulKinlan) [inline src/content/en/updates/posts/2016/04/chrome-51-deprecations.md]: The image path is wrong. It should be

/web/updates/images/2016/04/chrome-51-deprecations/deps-rems.png

**Comment by PaulKinlan** [comment]
Looks good other than the image url.

**Comment by PaulKinlan** [comment]
@jpmedley I've just fixed the image URL on this PR.  Will check to see if it appears and then merge.

**Comment by jpmedley** [comment]
Thanks!

Joe Medley | Technical Writer, Chrome DevRel | jmedley@google.com |
 816-678-7195
*If an API's not documented it doesn't exist.*

**Pull request merged**